hicexplorer/hicInfo.py

```diff
@@ -10,8 +10,8 @@ def parse_arguments(args=None):
                                                   'number of elements, sum of elements, etc.'))
 
     parser.add_argument('--matrices', '-m',
-                        help='matrices to add. Must have the same shape.',
-                        metavar='.h5 of .npz file format',
+                        help='The matrix (matrices) to get information about.',
+                        metavar='HiCExplorer supports: h5, npz, dekker and lieberman file format.',
```

Comment:
Here we have to be careful. What I call dekker format are the matrices produced by Job Dekker about 2-3 years ago. I don't know what are they using now. For Lieberman, we mean the matrices produced in the Lieberman publications from 2015.

Reply:
Should we set the formats `dekker` and `lieberman` to `deprecated`?

Reply:
I would deprecate this and go forward with Cooler.

Reply:
I know that people use that. I would not deprecate it at the moment, just point out that this refers to the format used by Job Dekker and Erez Lieberman Aiden and that they may change in the future.

Reply:
@fidelram I think we should indicate that this functionality will be removed at some point. It will simply the code a lot, especially with the cooler format that will be coming soon. I would propose an additional tool that converts from X to cooler. With such a tool people can still use the older formats.

Reply:
I will merge and take care of this afterwards.